src/WeightVectors.jl

```diff
@@ -1,5 +1,12 @@
 module WeightVectors
 
+# Use the README as the module docs
+@doc let
+    path = joinpath(dirname(@__DIR__), "README.md")
+    include_dependency(path)
+    read(path, String)
+end WeightVectors
+
```

Comment:
This is done automatically.

Reply:
yes, right, but this is a way to use the docstring into the documentation, is there another way?